code/src/styles/moviedetails.css

```diff
@@ -0,0 +1,60 @@
+/* Style for detailed movie page */
+.movie-details__background {
+    display: flex;
+    background-size: cover;
+    min-height: 100vh;
+    max-width: 100vw;
+}
+
+.movie-details__container {
+    display: flex;
+    align-items: flex-end;
+    padding: 50px;
+}
+
+.movie-details__poster {
+    border: 4px solid #f0f0f0;
+}
+
+.movie-details__info {
+    max-width: 450px;
+}
+
+.movie-details__title-rating {
+    display: inline-flex;
+    align-items: center;
+    margin: 20px 20px 0px 20px;
+}
+
+.movie-details__rating {
+    font-size: 17px;
+    margin-left: 10px;
+}
+
+.movie-details__overview {
+    margin: 0px 20px 20px 20px;
+    line-height: 1.2;
+}
+
+/* Media queries - responsive styling */
+
+@media (max-width: 767px) {
+    .movie-details__background {
+        flex-direction: column;
+        align-items: center;
+    }
+
+    .movie-details__container { 
+        flex-direction: column;
+        align-items: center;
+        padding: 10px;
+        margin-top: 40%;       
+    }
+}
+
+@media (max-width: 320px) {
+
+    .movie-details__poster {
+        width: 300px;
+    }
+}
```

Comment:
You've done a great job with the mediaqueries in the detailspage. It looks good in all sizes! I also really like the hover effect on the back link.